bot/cogs/sudo.py

```diff
@@ -44,8 +42,9 @@ def get_uptime(self) -> str:
         else:
             return f"{hours} hr, {minutes} mins, and {seconds} secs"
 
+    @staticmethod
     async def is_owner(ctx: Context) -> t.Union[bool, None]:
-        if ctx.author.id in config.devs:
+        if ctx.author.id in devs:
             return True
         else:
```

Comment:
This else is unnecessary, either the first if statement will run, ending the function, or if not, you can simply do what's in else, the `else` will always get triggered only when the first if doesn't pass so we might as well not use it.